conf/interviewers_conf.py

```diff
@@ -0,0 +1,5 @@
+interviewers_name = "nilaya"
+interviewers_email = "nilaya@qxf2.com"
+intervierwers_designation = "Senior QA"
+interviewers_starttime = "11:00"
+interviewers_endtime = "15:30"
```

Comment:
@nilaya123 Why do we have two different conf files? One is login_conf and interviewers_conf. can't we make it to one?

Reply:
@Annapooraniqxf2 : Yes, when I pull branch , this interviewers_conf.py is not seen. Am not sure how come it is there ,It is not needed.